test/unit/dialects/mssql/query-generator.test.js

```diff
@@ -58,52 +58,124 @@ if (current.dialect.name === 'mssql') {
       });
     });
+
+    test('createSchema', () => {
+      expectsql(QueryGenerator.createSchema('mySchema'), {
+        mssql: 'IF NOT EXISTS (SELECT schema_name FROM information_schema.schemata WHERE schema_name = \'mySchema\' ) BEGIN EXEC sp_executesql N\'CREATE SCHEMA [mySchema] ;\' END;'
+      });
+    });
+
+    test('showSchemasQuery', () => {
+      expectsql(QueryGenerator.showSchemasQuery(), {
+        mssql: 'SELECT "name" as "schema_name" FROM sys.schemas as s WHERE "s"."name" NOT IN ( \'INFORMATION_SCHEMA\', \'dbo\', \'guest\', \'sys\', \'archive\' ) AND "s"."name" NOT LIKE \'db_%\''
+      });
+    });
+
+    test('versionQuery', () => {
+      expectsql(QueryGenerator.versionQuery(), {
+        mssql: "DECLARE @ms_ver NVARCHAR(20); SET @ms_ver = REVERSE(CONVERT(NVARCHAR(20), SERVERPROPERTY('ProductVersion'))); SELECT REVERSE(SUBSTRING(@ms_ver, CHARINDEX('.', @ms_ver)+1, 20)) AS 'version'"
+      });
+    });
+
+    test('renameTableQuery', () => {
+      expectsql(QueryGenerator.renameTableQuery('oldTableName', 'newTableName'), {
+        mssql: 'EXEC sp_rename [oldTableName], [newTableName];'
+      });
+    });
+
+    test('showTablesQuery', () => {
+      expectsql(QueryGenerator.showTablesQuery(), {
+        mssql: 'SELECT TABLE_NAME, TABLE_SCHEMA FROM INFORMATION_SCHEMA.TABLES;'
+      });
+    });
+
+    test('dropTableQuery', () => {
+      expectsql(QueryGenerator.dropTableQuery('dirtyTable'), {
+        mssql: "IF OBJECT_ID('[dirtyTable]', 'U') IS NOT NULL DROP TABLE [dirtyTable];"
+      });
+    });
+
+    test('removeColumnQuery', () => {
+      expectsql(QueryGenerator.removeColumnQuery('myTable', 'myColumn'), {
+        mssql: 'ALTER TABLE [myTable] DROP COLUMN [myColumn];'
+      });
+    });
+
+    test('quoteIdentifier', () => {
+      expectsql(QueryGenerator.quoteIdentifier("'myTable'.'Test'"), {
+        mssql: '[myTable.Test]'
+      });
+    });
+
+    test('getForeignKeysQuery', () => {
+      expectsql(QueryGenerator.getForeignKeysQuery('myTable'), {
+        mssql: "SELECT constraint_name = C.CONSTRAINT_NAME FROM INFORMATION_SCHEMA.TABLE_CONSTRAINTS C WHERE C.CONSTRAINT_TYPE = 'FOREIGN KEY' AND C.TABLE_NAME = 'myTable'"
+      });
+    });
+
+    test('getForeignKeyQuery', () => {
+      expectsql(QueryGenerator.getForeignKeyQuery('myTable', 'myColumn'), {
+        mssql: "SELECT constraint_name = TC.CONSTRAINT_NAME FROM INFORMATION_SCHEMA.TABLE_CONSTRAINTS TC JOIN INFORMATION_SCHEMA.CONSTRAINT_COLUMN_USAGE CCU ON TC.CONSTRAINT_NAME = CCU.CONSTRAINT_NAME WHERE TC.CONSTRAINT_TYPE = 'FOREIGN KEY' AND TC.TABLE_NAME = 'myTable' AND CCU.COLUMN_NAME = 'myColumn'"
+      });
+    });
+
+    test('getPrimaryKeyConstraintQuery', () => {
+      expectsql(QueryGenerator.getPrimaryKeyConstraintQuery('myTable', 'myColumn'), {
```

Comment:
@sushantdhiman This is a duplicate. There already exists a test for `getPrimaryKeyConstraintQuery` [here](https://github.com/sequelize/sequelize/blob/fc03e068861ae5164b1c9be32d07a0b012c4e441/test/unit/dialects/mssql/query-generator.test.js#L103-L107)